geth/notification/fcm/notifier.go

```diff
@@ -0,0 +1,52 @@
+package fcm
+
+import (
+	"github.com/NaySoftware/go-fcm"
+	"github.com/status-im/status-go/geth/notification"
+)
+
+// Notifier represents messaging provider for notifications.
+type Notifier struct {
+	firebaseClient
+}
+
+// NewNotifier Firebase Cloud Messaging client constructor.
+func NewNotifier(key string) *Notifier {
+	return &Notifier{fcm.NewFcmClient(key)}
+}
+
+// Notify preparation and send to the tokens list.
+func (p *Notifier) Notify(body interface{}, tokens ...string) error {
+	p.setPayload(&notification.Payload{
+		Title: "Status - new message",
+		Body:  "ping",
+	})
+
+	p.setMessage(body, tokens...)
+	_, err := p.firebaseClient.Send()
```

Comment:
If you embed `firebaseClient` and use it transparently, let's use `p.Send()` for consistency?

Reply:
Notifier has already Send() method so we cant just call `p.Send` to call embeded method `p.firebase.Send()`.